rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -110,39 +148,87 @@ class TestStepSelector {
 //		// startup complete ensures the index to be populated
 //		Display.getDefault.syncExec[populateViewIfEmpty] // is only run, if view is created on startup
 //	}
-
 	@Focus
 	def void setFocus() {
 		viewer.control.setFocus
 	}
 
 	@Inject
 	@Optional
+	def void updateView(@EventTopic(SELECTOR_TOPIC_UPDATE) Object data) {
+		logger.debug("updateView.")
```

Comment:
This is just a debug hint, that the method was called. I think rewriting the method name in other words does not contain any additional information